operations_test.go

```diff
@@ -614,6 +636,29 @@ func TestKillProcess(t *testing.T) {
 	}
 }
 
+func TestSystemTime(t *testing.T) {
+	initOperationFakes(t)
+	fakeGNOI.SystemTime = func(context.Context, *spb.TimeRequest, ...grpc.CallOption) (*spb.TimeResponse, error) {
+		return &spb.TimeResponse{
+			Time: 12345,
+		}, nil
+	}
+	dut := DUT(t, "dut_cisco")
+	op, err := dut.Operations().SystemTime()
+	if err != nil {
+		t.Fatalf("Operatation failed, err %v", err)
+	}
+	if op != time.Unix(0, 12345) {
```

Comment:
initialize the want value in the if-condition so it doesn't need to be repeated:

if want:= time.Unix(0, 12345); want != got {
    t.Fatalf("Operation failed, want %v got %v", want, got)
}